src/cloud_provider/aws/kubernetes/mod.rs

```diff
@@ -229,6 +232,25 @@ impl<'a> EKS<'a> {
 
         let format_ips =
             |ips: &Vec<String>| -> Vec<String> { ips.iter().map(|ip| format!("\"{}\"", ip)).collect::<Vec<_>>() };
+        let format_zones = |zones: &Vec<AwsZones>| -> Vec<String> {
+            zones
+                .iter()
+                .map(|zone| format!("\"{}\"", zone.to_string()))
```

Comment:
Maybe you can implement newly introduced traits, it will bring more clarity and can be reused from other places `pub trait ToTerraformString` or / and `pub trait ToHelmString ` https://github.com/Qovery/engine/blob/739e3d12e9aa7bbcbe58be754d7344d066663cc7/src/models.rs#L1216